core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala

```diff
@@ -75,7 +75,7 @@ object CassandraSpec {
 
   val fallbackConfig = ConfigFactory.parseString(s"""
       akka.loggers = ["akka.persistence.cassandra.SilenceAllTestEventListener"]
-      akka.loglevel = INFO
+      akka.loglevel = DEBUG
```

Comment:
Left at debug on purpose?

Reply:
Yeah as we have log capturing on all the tests